travis/python_pypi_upload.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Should we deprecate this file all over in favor of the new include method?

Reply:
Yes, I think both of the python bash files should be deprecated. Eventually, I'm assuming we'll figure out similar yamls for Twincat CI then we can get rid of all of the bash scripts in favor of the shared configuration yamls.

Reply:
A big benefit to keeping scripts is that we can actually execute them locally for testing (i.e., use them in isolation), whereas with the yaml files, the best method I've seen so far is copy/paste line-by-line into a terminal.

Reply:
Ahh, true. How about I change the deprecation warnings to only print when running on Travis?

Reply:
Seems reasonable!